relay-config/src/config.rs

```diff
@@ -1291,12 +1291,21 @@ pub struct CardinalityLimiter {
     ///
     /// Defaults to 180 seconds, 3 minutes.
     pub cache_vacuum_interval: u64,
+
+    /// Sample rate for Cardinality Limiter Sentry errors.
+    ///
+    /// Rate needs to be between `0.0` and `1.0`.
+    /// If set to `1.0` all cardinality limiter rejections will be logged as a Sentry error.
+    ///
+    /// Defaults to 1% (0.01).
+    pub error_sample_rate: f32,
```

Comment:
Not a blocker, but should we put this into global config instead? Would be nice to control it via automator.